install_config/install/advanced_install.adoc

```diff
@@ -201,9 +201,25 @@ All].
 
 |`*openshift_master_named_certificates*`
 .2+.^|These variables are used to configure xref:../../install_config/certificate_customization.adoc#install-config-certificate-customization[custom certificates] which are deployed as part of the installation. See xref:advanced-install-custom-certificates[Configuring Custom Certificates] for more information.
-
 |`*openshift_master_overwrite_named_certificates*`
 
+|`openshift_hosted_registry_cert_expire_days`
```

Comment:
The other variables in the table are `*variable*`. Yours are `varaible`. Should be consistent.